src/libguac/timestamp.c

```diff
@@ -33,9 +33,21 @@ guac_timestamp guac_timestamp_current() {
 
     struct timespec current;
 
-    /* Get current time */
+    /* Get current time, monotinically increasing.
+       guacd expects that a client sync timestamp <= last_sent_timestamp. 
+       When time moves backwards, when last_sent_timestamp can move back, 
+       violating the assumption of forward moving time.
+       Ex. Obsorved a Azure Linux instance, 
+       where CLOCK_REALTIME jumped back often, causing disconnects.
+     
+    Posix does not require CLOCK_MONOTONIC. 
```

Comment:
POSIX*

Why is the indentation of this part of the block comment different from the rest?